### ISSUE TITLE
Dark theme has some Black text that does not look right/good.

### ISSUE THREAD

**Issue by leoheck** [obelisk79/OpenTheme#85]
Hi @obelisk79, thanks for this theme it is awesome. 

I am noticing there is some text that is black on the Dark theme, but this is kind of off.  I am using the AppImage, so maybe it happens on this thing only. 

Title of the panels:

![image](https://github.com/obelisk79/OpenTheme/assets/1277920/4f8a23f3-dfc4-46d6-b799-36c7f10dd7bd)

![image](https://github.com/obelisk79/OpenTheme/assets/1277920/20008e2b-39d7-4ffd-aff1-e54f89437a6f)

Some Items on the File Browser

![image](https://github.com/obelisk79/OpenTheme/assets/1277920/23087f2d-1619-442a-8cf3-2fecce8f9887)

![image](https://github.com/obelisk79/OpenTheme/assets/1277920/71e2264f-8071-4bf3-91f8-fd35b6250655)

```
OS: Ubuntu 24.04 LTS (ubuntu:GNOME/ubuntu)
Word size of FreeCAD: 64-bit
Version: 0.22.0dev.38001 (Git) AppImage
Build type: Release
Branch: main
Hash: 1fc6da959b936a2999774a4fe5b4c4b12bf6ac8d
Python 3.11.9, Qt 5.15.13, Coin 4.0.2, Vtk 9.2.6, OCC 7.7.2
Locale: English/United States (en_US)
Installed mods: 
  * POV-Ray-Rendering
  * Assembly4 0.50.12
  * InventorLoader 1.4.0
  * ose-piping
  * OpenTheme 2024.6.22
  * SheetMetal 0.4.20
  * fasteners 0.5.24
  * asm4_disable_version_check_popup
  * Render 2024.6.5
  * OpticsWorkbench 1.0.18
  * lattice2 1.0.0
  * kicadStepUpMod
```

**Comment by leoheck** [comment]
This may be related, https://github.com/obelisk79/OpenTheme/issues/76

**Comment by leoheck** [comment]
Another place with black text.

![image](https://github.com/obelisk79/OpenTheme/assets/1277920/e2259564-ccfe-4d5f-84bc-740519fa621e)

**Comment by obelisk79** [comment]
Thank you for making the report. This is a valid issue. I recently made changes to try and correct for active/inactive objects in the tree view. As part of that process I had to remove part of the theme that colored text for everything at once. I thought I had added it back everywhere it was needed, but obviously not. 

Over the last couple of days I've been busy completing course work for a design certification I am working on. The final assignment was submitted last night and hopefully I can finally start chipping away at these remaining text problems.

**Comment by chuckwagoncomputing** [comment]
Also seems to happen on Windows:
![Screenshot 2024-07-03 090950](https://github.com/obelisk79/OpenTheme/assets/1531127/4bdefc69-1a47-4018-ae08-cd7da425fc65)

**Comment by leoheck** [comment]
Yeah, on all panel headers, I suppose. I did not put all of them, haha.

**Comment by chuckwagoncomputing** [comment]
> Yeah, on all panel headers, I suppose. I did not put all of them, haha.

Windows as in the inferior operating system from Redmond; I mentioned it because you thought that perhaps it was only on the AppImage.

**Comment by leoheck** [comment]
Ah, ok, my mistake. That's good that it does not affect my package type only.

**Comment by obelisk79** [comment]
I've been trying to sort this out, targeting the panel headers to change the text color has proven to be a challenge so far. But I'll get there.

**Comment by leoheck** [comment]
I would like to help you @obelisk79 , I just have o figure out how.

**Comment by leoheck** [comment]
@obelisk79 yay! fixed!
![image](https://github.com/obelisk79/OpenTheme/assets/1277920/a4184aab-50ca-44b1-90fe-0a20b0fb324f)

**Comment by leoheck** [comment]
yay this too!

![image](https://github.com/obelisk79/OpenTheme/assets/1277920/5880990b-2c53-4ff6-a6bc-b90e9a8851e4)

**Comment by leoheck** [comment]
here too

![image](https://github.com/obelisk79/OpenTheme/assets/1277920/f5156102-27ac-4ca9-91fc-5a4ecfea346c)

**Comment by leoheck** [comment]
@obelisk79 I made a PR to show you where my fixes are but let me know if I should change something or, 

The extra thing is that I updated the selected color with this, as it is on Light Theme.
A dark theme is not good when the colors are bright and vibrant as if the thing was under the sun.
![image](https://github.com/obelisk79/OpenTheme/assets/1277920/11bb2afa-4593-4af0-81b6-058c096fc717)

**Comment by leoheck** [comment]
Ah there is this window here still.
![image](https://github.com/obelisk79/OpenTheme/assets/1277920/03f4c4dc-a887-4798-800d-3300e4ffcd96)

**Comment by leoheck** [comment]
This one was fixed too.
